contract_api/testcases/unit_testcases/test_banner_service.py

```diff
@@ -0,0 +1,38 @@
+import unittest
+
+from contract_api.infrastructure.models import Banner, CTA
+from contract_api.infrastructure.repositories.ui_content_repository import UIContentRepository
+from contract_api.services.banner_service import BannerService
+
+
+class MyTestCase(unittest.TestCase):
+    def setUp(self):
+        repo = UIContentRepository()
+        repo.delete_banners()
+        banner_list = [
+            Banner(id=1, image="https://d16o4vcu292x9v.cloudfront.net/assets/snet/minecraftizing-service/hero.jpeg"
```

Comment:
lets not reference urls in the code. it should come from a config

Reply:
This is dummy data for test case @raamb.

Reply:
I have replaced the URL with dummy string. 

As discussed, We also created a ticket to maintain the configs in environment variables 
https://trello.com/c/aJDdyJap